.github/workflows/checkdb.yml

```diff
@@ -34,4 +35,13 @@ jobs:
           ./cluster --cluster_config ../../mainnet/singularity/cluster_config_template.json --service S3>> S3.log 2>&1 &
           sleep 3
           cd - && cd cmd/cluster
-          ./cluster --cluster_config ../../mainnet/singularity/cluster_config_template.json --check_db --check_db_rblock_to=400000
+
+          WEEK_DAY=$(date +%w)
+          date
+          if [ $WEEK_DAY -eq 5 ];then
```

Comment:
as discussed offline, should be one day earlier, thus should equal 6?

Reply:
ok